sdk/storage/storage-file-datalake/src/generated/src/models/index.ts

```diff
@@ -378,7 +378,7 @@ export interface PathCreateOptionalParams extends coreHttp.RequestOptionsBase {
   renameSource?: string;
   /**
    * A lease ID for the source path. If specified, the source path must have an active lease and
-   * the leaase ID must match.
+   * the lease ID must match.
```

Comment:
We need to edit the swagger file for typos in the generated code, or it will be re-do by re-generation. 😿

Reply:
That sounds good! Thank you @ljian3377 !

Reply:
I made an issue 🌞 let me know if this is ok: https://github.com/Azure/azure-sdk-for-js/issues/9028